monetdbe/_cffi/convert/__init__.py

```diff
@@ -48,34 +48,33 @@ class MonetdbTypeInfo(NamedTuple):
     numpy_type: np.dtype
     c_string_type: str
     py_converter: Optional[Callable]
-    null_value: Optional[Union[int, np.floating]]
```

Comment:
if you removed that to fix mypy complaining, numpy added new type annotations, and in master i fixed the type annotation for the null field (an optional `np.floatable` I believe)